universum/modules/artifact_collector.py

```diff
@@ -176,13 +173,13 @@ def set_and_clean_artifacts(self, project_configs: Configuration, ignore_existin
 
         if artifact_list:
             name = "Setting and preprocessing artifacts according to configs"
-            self.artifact_list = self.structure.run_in_block(self.preprocess_artifact_list,
-                                                             name, True, artifact_list, ignore_existing_artifacts)
+            self.structure.run_in_block(self.preprocess_artifact_list,
```

Comment:
If you do not need the return value, you should remove returning it from the function.